python/mxnet/gluon/contrib/rnn/conv_rnn_cell.py

```diff
@@ -210,8 +214,10 @@ def hybrid_forward(self, F, inputs, states, i2h_weight,
         i2h, h2h = self._conv_forward(F, inputs, states,
                                       i2h_weight, h2h_weight, i2h_bias, h2h_bias,
                                       prefix)
-        output = self._get_activation(F, i2h + h2h, self._activation,
-                                      name=prefix+'out')
+        if F is symbol:
+            output = self.activation(i2h + h2h, prefix+'out')
```

Comment:
why not put out in constructor?

Reply:
This is so that the prefix (which depends on time-step) can be consistent.